RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate to Typescript mergeSubscriptionRooms and findSubscriptionsRooms

## Proposed changes

Migrate to Typescript mergeSubscriptionRooms and findSubscriptionsRooms

## Issue(s)	


## How to test or reproduce

After you open a room or the app, the tasks are merged automatically.
So just open the app or a room and check if the conversations are working properly.


## Screenshots

## Types of changes




## Checklist



## Further comments
